packages/runner/src/cell.ts

```diff
@@ -433,8 +433,8 @@ export class CellImpl<T> implements ICell<T>, IStreamable<T> {
     if (!space) {
       throw new Error(
         "Cannot create cell link: space is required.\n" +
-          "This can happen when closing over (opaque) cells in a lift or derive.\n" +
-          "Instead pass those cells into the lift or derive directly as Cell<> inputs.\n",
+          "This can happen when manually calling .get() on closed-over cells.\n" +
```

Comment:
probably other accesses of cells as well -- but not sure this case is ever triggered anymore